src/common/models/user_sets.ts

```diff
@@ -564,11 +563,11 @@ export function parseUserSet(user_set: ParseableUserSet) {
 }
 
 /**
- * Merge a ProblemSet, a DBUserSet and a CourseUser and return a UserSet. Note: if the
+ * Merge a ProblemSet, a DBUserSet and a User and return a UserSet. Note: if the
  * arguments are not related in the database (based on primary and foreign keys), a MergeError is
- * thrown.  in that the result is a MergedUserSet with overrides taken from the UserSet.
+ * thrown.  The result is a UserSet with overrides taken from the UserSet.
```

Comment:
This comment is confusing.  The UserSet is overridden by the same UserSet?  Looking at the code it seems that the default values from the `set.set_params` are used and those are overriden by values from the `db_user_set`.  Although it is a bit odd to use the terminology of override in this situation.  It seems that the effect is that the users values are used and the missing values are filled from the defaults.  So I would say `The result is a UserSet created with values from the DBUserSet and missing values taken from the ProblemSet.`